python/main.py

```diff
@@ -271,4 +276,13 @@ def select_join_items():
 
     conn.close()
 
-    return {"items": items_list}
\ No newline at end of file
+    return {"items": items_list}
+
+def delete_item(item_id):
+    conn = sqlite3.connect(db/"items.db")
+    cur = conn.cursor()
+
+    cur.execute('DELETE FROM items WHERE id = ?', (item_id,))

```

Comment:
This is totally depends on your intention and design but if you want to delete category when item is deleted, you can.
> But this command only removes the column from the items table, not the categories table...

One option is get category_id of the target item to delete and if there are no items which use the same category, you can delete it from category table.

In practice, category is usually defined by service provider (for example Mercari), and user can't define own category. Therefore, we don't have to care about category deletion when item is deleted.

Reply:
Thank you for your advice!
I could implement delete API maybe!!